spec/factories.rb

```diff
@@ -1,4 +1,8 @@
 FactoryBot.define do
+  factory :membership do
+    group { Group.first || association(:group) }
```

Comment:
this doesn't sound like a good idea because it can get quite nasty to debug, please just keep the association